torch/distributions/constraints.py

```diff
@@ -565,6 +577,28 @@ def check(self, value):
                             for v, constr in zip(vs, self.cseq)], self.dim)
 
 
+class _LogicalAnd(Constraint):
```

Comment:
Can we remove this constraint from this PR? We can add it later if we need it, but not sure if this is much simpler than `constraint1.check(value) & constraint2.check(value)`. If the motivation is to separate `symmetric` and `positive_definite` constraints, I'm not sure if it is very straightforward (see above). Do we need these constraint checks for non-symmetric matrices in your implementation?

Reply:
It is my bad. I was working on other branch and accidentally added the code to this PR. Fixed it.